types/feathersjs__authentication-client/index.d.ts

```diff
@@ -0,0 +1,64 @@
+// Type definitions for @feathersjs/authentication-client 1.0
+// Project: http://feathersjs.com/
+// Definitions by: Abraao Alves <https://github.com/AbraaoAlves>, Jan Lohage <https://github.com/j2L4e>
+// Definitions: https://github.com/feathersjs-ecosystem/feathers-typescript
+
+import './feathersjs__feathers';
+
+export default function feathersAuthClient(config?: FeathersAuthClientConfig): () => void;
+
+export interface FeathersAuthClientConfig {
+    storage?: Storage;
```

Comment:
nit: Prefer doc comments here, even if a copy of what's available on other sites, to provide autocomplete self-documentation